modules/ROOT/pages/single-sign-on-with-oidc.adoc

```diff
@@ -256,20 +256,20 @@ If the IdP declares a redirect URI, it might override the target URL (`redirect_
 
 == Configure logout behaviour
 
-If your Bonita platform is configured to manage authentication over OIDC, when users log out of Bonita Portal, they do not log out of the OIDC Identity Provider (IdP).
+If your Bonita platform is configured to manage authentication over OIDC, when users log out of Bonita Runtime, they do not log out of the OIDC Identity Provider (IdP).
```

Comment:
above, with CAS,  it was mentionnes that you do get log out. does the system act differently depending on the identity provider?